indexer/config/presets.go

```diff
@@ -4,73 +4,109 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// in future presets can just be onchain config and fetched on initialization
-// Mapping of l2 chain ids to their preset chain configurations
-var presetConfigs = map[int]ChainConfig{
-	// OP Mainnet
+type Preset struct {
+	Name        string
+	ChainConfig ChainConfig
+}
+
+// In the future, presets can just be onchain config and fetched on initialization
+
+// Mapping of L2 chain ids to their preset chain configurations
+var presetConfigs = map[int]Preset{
 	10: {
-		L1Contracts: L1Contracts{
-			OptimismPortalProxy:             common.HexToAddress("0xbEb5Fc579115071764c7423A4f12eDde41f106Ed"),
-			L2OutputOracleProxy:             common.HexToAddress("0xdfe97868233d1aa22e815a266982f2cf17685a27"),
-			L1CrossDomainMessengerProxy:     common.HexToAddress("0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1"),
-			L1StandardBridgeProxy:           common.HexToAddress("0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"),
-			LegacyCanonicalTransactionChain: common.HexToAddress("0x5e4e65926ba27467555eb562121fac00d24e9dd2"),
+		Name: "Optimism Mainnet",
+		ChainConfig: ChainConfig{
+			L1Contracts: L1Contracts{
+				AddressManager:                  common.HexToAddress("0xdE1FCfB0851916CA5101820A69b13a4E276bd81F"),
```

Comment:
holding open just until I can re-check these

cc @roninjin10 would appreciate a second eye on these contract addresses

Reply:
If you want piece of mind you can validate them in a script against these addresses here. These addresses are completely generated from our artifacts. https://github.com/ethereum-optimism/optimism/pull/6301/files#diff-b820215d5bd670fe39c050ac07fa2c98e3150b080033fe34b290981387da9207R126

Reply:
triple checked everything